examples/basic-queue-operations/Ballerina.toml

```
@@ -0,0 +1,10 @@
+[package]
+org = "ballerinax"
```

Use WSO2 as the organization in the examples. Check similar places.

```suggestion
org = "wso2"
```

---

Updated.